### PR TITLE
ENH: stats: Implement the survival function for pareto.

### PR DESCRIPTION
**Pull request by WarrenWeckesser**

### CODE REVIEW

File: scipy/stats/tests/test_distributions.py [hunk 1]
```diff
@@ -621,6 +621,10 @@ def test_stats(self):
             assert_allclose(s, (2*5.5/1.5) * np.sqrt(2.5/4.5))
             assert_allclose(k, 6*(4.5**3 + 4.5**2 - 6*4.5 - 2)/(4.5*1.5*0.5))
 
+    def test_sf(self):
+        p = stats.pareto.sf(1e9, 2, loc=0, scale=1.5)
+        assert_allclose(p, 2.25e-18)
```
Comment (Eric89GXL) [inline scipy/stats/tests/test_distributions.py]: Might be worth mentioning where you got the veridical value

**Review by larsoner — COMMENTED** [review]
LGTM. Not that familiar with the stats stuff, but I assume these changes can be so minimal because documentation and such gets automatically generated, is that right?

File: scipy/stats/tests/test_distributions.py (hunk 1 above)
Comment (WarrenWeckesser) [inline scipy/stats/tests/test_distributions.py]: "Veridical"--now there's a ten dollar word!  It is just (1.5/1e9)**2.  In general, it is (scale/x)**b.

**Comment by WarrenWeckesser** [comment]
> Not that familiar with the stats stuff, but I assume these changes can be so minimal because documentation and such gets automatically generated, is that right?

Yes, this change just overrides the default method (which uses 1 - cdf(x, ...) to compute the survival function).  A change like this could even be called a bug fix rather than an enhancement, if you think that `sf(1e9, 2, loc=0, scale=1.5)` returning 0 instead of 2.25e-18 is a bug.

File: scipy/stats/tests/test_distributions.py (hunk 1 above)
Comment (WarrenWeckesser) [inline scipy/stats/tests/test_distributions.py]: I pushed a new version that shows where 2.25e-18 comes from.

**Review by larsoner — APPROVED** [review]

**Comment by ev-br** [comment]
LGTM, merged. Thanks Warren, Eric

**Pull request merged**